olclient/openlibrary.py

```diff
@@ -196,8 +198,15 @@ def editions(self):
                 """
                 url = f'{self.OL.base_url}/works/{self.olid}/editions.json'
                 try:
-                    r = self.OL.session.get(url)
-                    editions = r.json().get('entries', [])
+                    r_json: Dict = self.OL.session.get(url).json()
+                    editions: List = r_json.get('entries', [])
```

Comment:
The same argument as above: `List[Any]`. Also, if we know what type of elements we are getting in here we could use that instead.